addons/web/static/src/views/calendar/calendar_common/calendar_common_popover.scss

```diff
@@ -69,3 +69,7 @@ $o-cw-popup-avatar-size: 16px;
         }
     }
 }
+
+.o_cw_body {
+    max-width: 328px;
+}
```

Comment:
I see my comment has not been published. 😨 

Since you duplicate the style from `calendar`, why not removing it in `calendar` since `web` module is in the dependencies of `calendar` module. 🤷🏼

Reply:
Yeah, my bad!
Thanks for mentioning it. :slightly_smiling_face: